gateway/conf/nginx.conf.liquid

```diff
@@ -122,7 +122,32 @@ http {
 
     access_log {{ access_log_file | default: "/dev/stdout" }} time;
 
-    listen {{ port.apicast | default: 8080 }};
+    {%- assign http_port = port.apicast | default: 8080 %}
+    {%- assign https_port = env.APICAST_HTTPS_PORT %}
+
+    {% if http_port != https_port -%}
+      listen {{ http_port }};
+    {% endif %}
+
+    {% if https_port -%}
+    listen {{ https_port }} ssl;
+
+    {%- assign https_certificate = env.APICAST_HTTPS_CERTIFICATE -%}
+    ssl_certificate {% if https_certificate or https_certificate != empty -%}
```

Comment:
I'm not sure I understand this. Shouldn't it be an `and` instead of an `or` ?
Does an empty string pass the first part of the `or` in Liquid?

Reply:
I was trying to work around an issue with the liquid library. `{% "somestring" == empty %}` crashes. Will remove the `or ` part as that would still not execute when using magic tables like in other parts (where we have to check for == empty).